gulpfile.js

```diff
@@ -12,7 +12,7 @@ gulp.task('lint', function() {
 });
 
 gulp.task('test', function() {
-  return gulp.src('./test/test.js', {read: false})
+  return gulp.src('test.js', {read: false})
```

Comment:
Great work here. That was a frustration with the assignment and you did it by scratch. Great work!